misaligned.cpp

```diff
@@ -1,21 +1,39 @@
 #include <iostream>
 #include <assert.h>
 
+void checkPairNumberValidity(int pair_no)

```

Comment:
Test code is part of production code. Asserts are for testing if the production code works as expected. here Asserts are part of production code. Please segregate test code and production code.